synapse/tests/utils.py

```diff
@@ -955,11 +955,11 @@ def ne(self, x, y):
         '''
         self.assertNotEqual(x, y)
 
-    def true(self, x):
+    def true(self, x, msg=None):
```

Comment:
Is anyone opposed to me adding these on all the wrappers over the `unittest` `assert*` functions that take it? It's a nice way to give an explanation when an assertion fails.  `AssertionError: 1 != 2 : on foo:baz` vs `AssertionError: 1 != 2` (wat)